### DISCUSSION TITLE
orca slicer setup with bassaminanator

### DISCUSSION THREAD

**Discussion by dasb00ter** [bassamanator/Sovol-SV06-firmware#162] [Q&A] [answered]
I've klipperized my printer with this wonderful repo. I've added a few printer profiles and config settings profiles to orca slicer but they seem to call a different start macro(s). Seems as if my first hurdle is start gcode as I get errors that clearly demonstrate things are not in place to start a print with these profiles. Can I or should I try and get the start gcode to call bassaminators start macro with or without kamp with these orca slicer imports or is that a NO NO. I'm guessing all of this will be done for me if I import the SS or prusa bassaminator configs, but I would be using a different slicer. Moving forward though I just don't know what I can change and not break things with something like [https://github.com/3DPrintDemon/Demon_KLIPPER_Essentials ](url). Does anybody have a complete set up I can examine that uses orca?

**Comment by bassamanator** [comment, marked as answer]
Check the adjust your slicer section. This applies to any klipper configuration or slicer that you're using. If you're using OS, use the same start gcode as PS. I've already responded about KAMP separately.

`PRINT_START` is a macro contained within my config. That macro is *called* from the start gcode section of the slicer.

**Reply by dasb00ter** [reply]
Thanks. I appreciate your help. I did read those references and I believe OS is not mentioned(PS and SS are).  This question was posed to other users who use OS which I have read that you do not lol (that's how much I read). Even saw someone joking about you using mechanical gantry leveling on Reddit somewhere. In any case I can see at least one user trying to use OS and Kamp and not getting the print to end properly.I was under the impression that the way slicers use variables and call the start macro may be different. Again thanks for your help.

**Comment by bassamanator** [comment]
Since you've done the readings...

It's true that I don't use OS, and based on the fact that OS is not mentioned on ellis3dp.com, he doesn't either. I have nothing against OS in principle, it's just that I can't stand the UI (user interface) and I care about UI and UX (user experience) a lot, because that's part of my job outside the 3d printing world. I wish the project great success regardless, anything that adds variety to the 3d printing world is just fine by me.

I have written my configuration so that the user need not be concerned with editing code, so you don't have to edit any of the macros (unless you want to). *If* you would like to make some setting changes, such as enabling KAMP, or disabling the filament sensor, you need only change a `0` to a `1` and a `1` to a `0`, respectively.

Do not edit the start gcode (the slicer portion), unless [this](https://github.com/bassamanator/Sovol-SV06-firmware?tab=readme-ov-file#line-purge).

**Comment by dasb00ter** [comment]
> Since you've done the readings...


Well I mean I try. Your name and input comes up frequently. I'm back and forth with Ellis and Reddit and Printables and your GitHub and FDMprinting etc. 
I feel like many new printers like myself are easily swayed and the YouTube videos and tutorials promise "tune your printer in one hour using this or that slicer built in tuning system" etc. Also OS apparently has some great defaults for marlin Sv06 / + and there is the feeling u kind of made it when you get a quality print(even though your just starting lol). 

I'll go back to PS and retry that based on your experience or SS from February if that works with your profiles

**Reply by bassamanator** [reply]
Go with PS until SS gets its next update.

There is a very simple reason everyone, particularly n00bs, should use the Ellis's print settings (the ones that are found on this repo), is because there's a changelog for those print settings going back roughly 2 years. They are a product of many slow adjustments along with a lot of testing. The minute someone shows me some other settings with a changelog going back over 2 years, I'll switch to using them.

**Reply by dasb00ter** [reply]
So I am currently using the ellis profiles and I certainly have some printer tuning to do it looks like but I am happy to not have my extruder crashing into my buildplate or carving a line in the PEI plate.

The ellis print settings give descriptions but are a little hard to follow. Decorative vs print it forward Voron parts? I know my printer, if the speed is reduced enough and with the right presets(stock sovol prusa/orca marlin profiles)  can be babied into producing a pretty nice print(it takes forever but such is life). While I continue to tune the printer is one of these profiles more suitable to producing slower more quality prints so I can keep printing...which kind of keeps the moral up

**Reply by bassamanator** [reply]
Go to Discussions > Q&A, and check my posts these print settings.

**Reply by dasb00ter** [reply]
I think I read that posts already. Is it this one:  [https://github.com/bassamanator/Sovol-SV06-firmware/discussions/14/](url)

Is there more direction there bc this doesnt really answer my question. Also the profiles imported seem different so can you confirm I havent made an error importing them
![Screenshot (28)](https://github.com/bassamanator/Sovol-SV06-firmware/assets/36972116/51e6da57-9f18-401f-8892-dfabb28acc7d)

**Reply by bassamanator** [reply]
PIF vs decorative is explained in the video. These print settings take care of the speed for you, as long as your machine can achieve the speeds/accels listed in the speed section.

**Reply by dasb00ter** [reply]
Thanks for getting back to me. I continue to struggle with print quality using the Ellis print settings. I've moved over to super slicer as I think I could find more info about your use of profiles with that. My best guess is under extrusion as there are largish random gaps and the corners are not very good(maybe that's pressure assist related) but I don't think this is a "fix my print" discussion space.  I had to revert to the old marlin firmware and the sovol orca slicer profile to get a mother's day print done. The print is decent with that but incredibly conservative and slow. 

My thoughts are to try and transfer the start/end gcode from prusa/super slicer into orca and try and use that workable solvo slow profile and then at least I can print things and then go from there. 

Just to clarify I've squared the gantry,done esteps, leveled the bed(silicon mod), I've tested the input shaper for X but haven't entered the values from the chart yet. The printer looks and sounds good to me, no rattle.I continue to learn things but I'm just surprised BC it seems like everyone just sets up from this GitHub and they are printing aerospace tolerance objects at high speed

**Reply by bassamanator** [reply]
Not sure if i've said this here before, SS has been updated recently but still needs catching up with PS. At this point, I'm still recommending PS. When SS catches up, I'll be switching *back* to SS.

If you want to slow down your print speeds easily, simply adjust Max speeds here:
![image](https://github.com/bassamanator/Sovol-SV06-firmware/assets/61985779/eb82553e-6013-4076-89cd-a75cd55dcf69). You can also tune individual values on the same page as well.

To improve print quality, in no particular order:
1. You need to measure resonances for X and Y.
2. Do pressure advance/input shaping 
3. Calibrate e-steps.
4. Adjust flow rate of your filament.
5. Print only in PLA/+ for now.
6. Get a proper first layer.
7. Only use settings that are already known to be good.
